src/widget/wsearchrelatedtracksmenu.cpp

```diff
@@ -3,6 +3,20 @@
 #include "track/track.h"
 #include "util/qt.h"
 
+namespace {
+
+constexpr double kRelativeBpmRange = 0.06; // +/-6 %
+
+inline int bpmLowerBound(double bpm) {
```

Comment:
I'd prefer to add a direct way to search within a percentage (or absolute) range for numeric search operators than have this calculated in a GUI class. Can you add a TODO comment for that?

Reply:
I don't think that we need to add an explicit TODO here. It doesn't help. Otherwise, I would need to add TODO to all code saying "this code will probably be reusable, please extract it when needed" ;)

The constants and functions are already separated and extracting this functionality should be straightforward.

Reply:
Only the query is composed inline to avoid formatting the numbers twice.

Reply:
> The constants and functions are already separated and extracting this functionality should be straightforward.

Yes, but my point is this shouldn't be anywhere in wsearchrelatedtracksmenu.cpp.